impeller/golden_tests/golden_playground_test_mac.cc

```diff
@@ -77,7 +81,7 @@ std::string GetGoldenFilename() {
   return GetTestName() + ".png";
 }
 
-bool SaveScreenshot(std::unique_ptr<testing::MetalScreenshot> screenshot) {
+bool SaveScreenshot(std::unique_ptr<testing::Screenshot> screenshot) {
```

Comment:
(non-blocking) Could this just be a reference to a screenshot instead of a pointer?

Reply:
Yep, it could have been.  In this case taking ownership of the screenshot or not is moot since no one ever uses it after it's saved.  The minor downside of that is that in c++ you don't know from the call site if you are working with references or not like you do with rust.  So keeping things as smart pointers avoids the situation where you have to make sure you aren't accidentally introducing an implicit copy.

Reply:
(if I didn't want to take ownership, I would probably have made a reference to a unique_ptr instead of using a reference to a Screenshot for the above reason.)